### PR TITLE
11.0 mig stock partner lot

### PR DESCRIPTION
**Pull request by agungrachmatullah**
Migrating and adding Unit Test

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

And also check Travis status.

File: stock_partner_lot/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    """Inherit res.partner to add some fields and create
+    computation of function for quant"""
+    _inherit = 'res.partner'
+
+    quant_ids = fields.One2many('stock.quant', 'owner_id',
+                                string='Owned products')
+    quant_count = fields.Integer('Owned Products',
+                                 compute='_compute_quant_count')
+
+    @api.multi
```
Comment (rousseldenis) [inline stock_partner_lot/models/res_partner.py:18]: Please depends on quant_ids

File: stock_partner_lot/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    def _compute_quant_count(self):
+        """compute the quant each of partnet"""
```
Comment (rousseldenis) [inline stock_partner_lot/models/res_partner.py]: Maybe 'Compute the quants length for each partner'

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

File: stock_partner_lot/models/res_partner.py (hunk 1 above)
Comment (agungrachmatullah) [reply stock_partner_lot/models/res_partner.py:18]: Oh my, yes the quant_count should updated when the value of quant_ids has changed, ill revise it and the description, thanks @rousseldenis

**Review by agungrachmatullah — COMMENTED** [review]
depends updated

**Review by rousseldenis — APPROVED** [review]

File: stock_partner_lot/tests/test_stock_partner_lot.py [hunk 3]
```diff
@@ -17,8 +17,9 @@ def setUp(self):
         self.product_2 = self.env.ref('product.product_product_6')
 
         res_users_purchase_user = self.env.ref('purchase.group_purchase_user')
-        user = self.env['res.users'].with_context\
-        ({'no_reset_password': True, 'mail_create_nosubscribe': True})
+        user = self.env['res.users'].with_context(
+            {'no_reset_password': True, 'mail_create_nosubscribe': True}
+        )
         self.purchase_user = user.create({
```
Comment (bimajatiwijaya) [inline stock_partner_lot/tests/test_stock_partner_lot.py:21]: this code unused i think `self.purchase_user`.

Comment (bimajatiwijaya) [reply stock_partner_lot/tests/test_stock_partner_lot.py:21]: If you plan to use this user to create data you should add `...sudo(self.purchase_user).create...` at [L41](https://github.com/agungrachmatulah/stock-logistics-warehouse/blob/01cc9dcae79ff62edf4e7bca57be9a425060ffc1/stock_partner_lot/tests/test_stock_partner_lot.py#L41)

**Review by bimajatiwijaya — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Hello!
Great work! I was looking for exactly this module. The problem is however on latest runbot and my odoo instance owner_id is always null on stock.move.line and stock.quant. I would have expected to be owner_id the partner of the picking.

So this module won't have any effect.

Can you confirm this is working for you as expected and owner_id is set on your move lines/quants? If so, do you have special configuration?

Thanks for your insights!

**edit: nevermind found the configuration. Should have read the module readme..**
**edit2: Sorry the spam:
Does anybody have an approach to just show the quants sent to a partner?**

**Comment by rousseldenis** [comment]
@agungrachmatulah Can you attend comments ?

**Comment by rousseldenis** [comment]
@agungrachmatulah

**Comment by rousseldenis** [comment]
@agungrachmatulah

**Comment by rousseldenis** [comment]
Closing this as it's quite old. Feel free to reopen it if needed

**Pull request closed without merging**